internal/devconfig/configfile/file.go

```diff
@@ -80,36 +78,15 @@ type Stage struct {
 	Command string `json:"command"`
 }
 
-const DefaultInitHook = "echo 'Welcome to devbox!' > /dev/null"
-
-func DefaultConfig() *ConfigFile {
-	cfg, err := loadBytes([]byte(fmt.Sprintf(`{
-  "$schema": "https://raw.githubusercontent.com/jetpack-io/devbox/main/.schema/devbox.schema.json",
-  "packages": [],
-  "shell": {
-    "init_hook": [
-      "%s"
-    ],
-    "scripts": {
-      "test": [
-        "echo \"Error: no test specified\" && exit 1"
-      ]
-    }
-  }
-}
-`, DefaultInitHook)))
-	if err != nil {
-		panic("default devbox.json is invalid: " + err.Error())
-	}
-	return cfg
-}
-
 func (c *ConfigFile) Bytes() []byte {
 	b := c.ast.root.Pack()
 	return bytes.ReplaceAll(b, []byte("\t"), []byte("  "))
 }
 
 func (c *ConfigFile) Hash() (string, error) {
+	if c.ast == nil {
+		return cachehash.JSON(c)
+	}
```

Comment:
in what scenario would `c.ast` be `nil`? 


Instead, should we be building the ast in this if-block?

Reply:
plugin configs don't use hujson (but will in the future). Scope of this change is already too big to include that moving part.